src/cpr_sdk/pipeline_general_models.py

```diff
@@ -29,7 +29,7 @@ class BackendDocument(BaseModel):
     family_import_id: str
     family_slug: str
     publication_ts: datetime
-    date: Optional[str] = None  # Set on import by a validator
+    date: Optional[str] = None
```

Comment:
**suggestion:** Mark this as deprecated [1].

[1] >  I've also left the field itself so it can still be used if we have any more-legacy type code lying around

Reply:
Possibly use https://docs.pydantic.dev/latest/concepts/fields/#deprecated-fields, if we're on the right Pydantic version.

Reply:
Good idea